src/main/java/com/project/gamestore/controllers/UserController.java

```diff
@@ -0,0 +1,51 @@
+package com.project.gamestore.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.project.gamestore.dtos.UserDTO;
+
+import java.util.*;
+
+import com.project.gamestore.services.UserService;
+
+@RestController
+@RequestMapping("/users")
+public class UserController{
+
+    @Autowired
+    private UserService UserService;
```

Comment:
In Java each variable should be declared using lowercase letters in the beginning of the name.
`UserService UserService` -> `UserService userService`